sagemaker_neo_compilation_jobs/tensorflow2_resnet/sagemaker-neo-tf-resnet.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>Branding - please change all markdown occurrences of <code>Tensorflow</code> to the proper name/formatting of <code>TensorFlow</code> .  </p><p> </p><p>Please insert a separate PNG image into the notebook rather than including the image embedded in the notebook file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>